### PR TITLE
feat: launch

### PR DESCRIPTION
**Pull request by tomwisecodes**
This removes the pages for comparative judgment and the image alt text generator.

This also removes the feature flag opening aila up to the public and launching the project 🫨

### CODE REVIEW

File: apps/nextjs/src/app/home-page.tsx [hunk 1]
```diff
@@ -162,26 +160,19 @@ export default function HomePage({ featureFlag }) {
                 resources.
               </OakP>
 
-              {!userId || !featureFlag ? (
```
Comment (codeincontext) [inline apps/nextjs/src/app/home-page.tsx:165]: This looks like a functional change if the userId doesn't immediately load. I don't know if that can happen

File: apps/nextjs/src/components/Footer.tsx [hunk 2]
```diff
@@ -73,9 +70,6 @@ const Footer = ({ featureFlag }: FooterProps) => {
                           }
                         }}
                         href={tool.href}
-                        disabled={
```
Comment (codeincontext) [inline apps/nextjs/src/components/Footer.tsx:76]: What about `!tool.href`? Is href a required field now?

File: apps/nextjs/src/app/home-page.tsx (hunk 1 above)
Comment (tomwisecodes) [reply apps/nextjs/src/app/home-page.tsx:183]: This was to do with creating a button saying coming soon or not weather the user was logged in

File: apps/nextjs/src/components/Footer.tsx (hunk 2 above)
Comment (tomwisecodes) [reply apps/nextjs/src/components/Footer.tsx:80]: The types allowed it to build so I assumed this was fine

**Pull request merged**